docs/reference/datatiers.asciidoc

```diff
@@ -127,5 +122,6 @@ If you set the tier preference to `null`, {es} ignores the data tier roles durin
 {ilm-init} automatically transitions managed
 indices through the available data tiers using the <<ilm-migrate, migrate>> action.
 By default, this action is automatically injected in every phase.
-You can explicitly specify the migrate action to override the default behavior,
-or use the <<ilm-allocate, allocate action>> to manually specify allocation rules.
+You can explicitly specify the migrate action to disable automatic migration,
```

Comment:
I think it'd be useful to say "specify the migrate action with 'enabled': false", because just manually specifying the migrate action doesn't actually disable it.

Reply:
Will do! I was already on the fence about exactly that, so thanks for giving me a reason to choose one way over the other.

Reply:
f61533c